core/src/sidebar/common.tsx

```diff
@@ -0,0 +1,8 @@
+export interface menu {
+	path: string;
+	title: string;
+	type: React.ReactNode;
+	icon: { outline: React.ReactNode; solid: React.ReactNode };
```

Comment:
I think this should extend via intersection with Moai's icon type.

The reason is some icons in `@banhmi/icon` not have both outline and solid variants. So I don't want to pass prop like `icon={{ outline: SomeIcon, solid: SomeIcon }}`, just `icon={SomeIcon}`